build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -267,9 +267,9 @@ public static BuildTargetResult GenerateMSbuildPropsFile(BuildTargetContext c)
             msbuildProps.AppendLine($"    <HostVersion>{hostVersion.LatestHostPolicyVersion.WithoutSuffix}</HostVersion>");
             msbuildProps.AppendLine($"    <HostResolverVersion>{hostVersion.LatestHostFxrVersion.WithoutSuffix}</HostResolverVersion>");
             msbuildProps.AppendLine($"    <HostPolicyVersion>{hostVersion.LatestHostVersion.WithoutSuffix}</HostPolicyVersion>");
-            msbuildProps.AppendLine($"    <BuildNumberMajor></BuildNumberMajor>");
-            msbuildProps.AppendLine($"    <BuildNumberMinor></BuildNumberMinor>");
-            msbuildProps.AppendLine($"    <PreReleaseLabel></PreReleaseLabel>");
+            msbuildProps.AppendLine($"    <BuildNumberMajor>{hostVersion.LatestHostBuildMajor}</BuildNumberMajor>");
+            msbuildProps.AppendLine($"    <BuildNumberMinor>{hostVersion.LatestHostBuildMinor}</BuildNumberMinor>");
```

Comment:
Are these versions right? HostVersion is already at `1.0.2`, is that what we want here?